api/agent/deployer/deployer.go

```diff
@@ -62,10 +61,3 @@ func (c *Client) Machine(tag names.MachineTag) (*Machine, error) {
 		client: c,
 	}, nil
 }
-
-// ConnectionInfo returns all the address information that the deployer task
-// needs in one call.
-func (c *Client) ConnectionInfo() (result params.DeployerConnectionValues, err error) {
-	err = c.facade.FacadeCall(context.TODO(), "ConnectionInfo", nil, &result)
-	return result, err
-}
```

Comment:
Dead code?

Reply:
yeah. left the api server endpoint in place just in case